test/parallel/test-tls-js-stream.js

```diff
@@ -1,43 +1,42 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
 
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-var tls = require('tls');
+const tls = require('tls');
 
-var stream = require('stream');
-var fs = require('fs');
-var net = require('net');
+const stream = require('stream');
+const fs = require('fs');
+const net = require('net');
 
-var connected = {
+const connected = {
```

Comment:
The `connected` object and the calls to `connected.server++` and `connected.client++` can be removed here also.